generators/traefik/traefk_test.go

```diff
@@ -0,0 +1,166 @@
+package traefik
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/stretchr/testify/require"
+
+	"github.com/kubeshop/kusk/options"
+	"github.com/kubeshop/kusk/spec"
+)
+
+func TestTraefik(t *testing.T) {
+	var testCases = []struct {
+		name    string
+		options options.Options
+		spec    string
+		res     string
+	}{
```

Comment:
For the tests here, they are based on the tests used in the nginx_ingress generator package

Feel free to lift them @Tarick